projects/arlas-components/src/lib/components/powerbars/powerbars.component.ts

```diff
@@ -194,6 +194,15 @@ export class PowerbarsComponent implements OnInit, OnChanges, AfterViewInit {
   public NUMBER_FORMAT_CHAR = NUMBER_FORMAT_CHAR;
 
   public constructor(private colorService: ArlasColorService) {
+    this.colorService.changekeysToColors$.subscribe(() => {
+        this.powerBarsList.forEach(p => {
+          if (this.useColorService) {
+            const rgbaColor = tinycolor.default(this.colorService.getColor(p.term, this.keysToColors,
+              this.colorsSaturationWeight)).toRgb();
+            p.color = 'rgba(' + [rgbaColor.r, rgbaColor.g, rgbaColor.b, 0.7].join(',') + ')';
```

Comment:
Maybe the 0.7 could be a variable, or there could be a method to do this color conversion for each powerbar?

Reply:
I kept the same code as the original. It's not the purpose of this PR to change that

Reply:
My comment was a suggestion to put this part of the code that is duplicated due to the PR in a method for example, to avoid diverging behaviors in the future. The idea of the variable is to avoid having a hard coded value, not necessarily to allow the user to edit that value, which I agree, is not the goal of the PR.